src/test/java/org/minions/devfund/benjamin/MovieTest.java

```diff
@@ -0,0 +1,53 @@
+package org.minions.devfund.benjamin;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * class to tes movie class.
+ */
+public class MovieTest {
+    /**
+     * Test set name function.
+     */
+    @Test
+    public void testSetName() {
+        Movie movie = new Movie();
+        String movieName = "BATMAN";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "BATMAN" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)